scripts/infra/cloud-instance.sh

```diff
@@ -34,7 +34,7 @@ ec2__launch() {
         echo "Failed to launch instance"
         exit 1
     else
-        local i=0
+        local i=1
```

Comment:
(joking) Ah but now this code waits for 55 seconds, not 12*5! Horrible!

Reply:
(can't stop myself) Ah and the last sleep is useless because it will not retry again before claiming defeat! Full 5 seconds wasted before telling the user their VM is not up!

Reply:
I cannot disagree with these comments. My only defense is that the PR makes it not worse :laughing: